e2e/test/scenarios/admin-2/sso/shared/group-mappings-widget.js

```diff
@@ -96,6 +96,8 @@ const addGroupsToMapping = (mappingName, groups) => {
         });
     });
   });
+
+  cy.realPress("{esc}");
```

Comment:
This fixes the JWT e2e spec by closing the popover, so it doesn’t cover a button that has to be pressed afterward.